components/ModDetails/ModDetails.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Card } from 'react-bootstrap'
+
+function ModDetails() {
+    return (
+        <>
+            <Card>
+                <Card.Header>
+                    <Card.Title>Details</Card.Title>
+                </Card.Header>
+                <Card.Body>
+                    <p>We provide a mod as a quality of life feature to use the flipper in minecraft directly.</p>
+                    <p>To use the mod you will need to link it to the website and verify your Minecraft account.</p>
```

Comment:
The mod does way more than just provide flips in game. Also you do NOT need to link and/or verify.
```suggestion
                    <p>Say goodbye to window switching. The cofl mode provides you with data directly in hypixel skyblock. You can access price paid, the ah flipper, median, lbin, bazaar buy and sell prices and more.</p>
                    <h3>Installation TLDR
                    <ol>
                    <li>Get and install forge 1.8.9 (if you don't have already watch the video below)</li>
                    <li>Download and put the mod into your mods folder</li>
                    <li>Launch minecraft and join skyblock</li>
                    <li>Get a list of commands with /cofl help or watch the <a href="https://www.youtube.com/watch?v=dv66Y_cra8U&list=PLDpPmxIcq9tAssQlyJMBlSmSg5JOpq699">YouTube playlist</a> explaining them</li>
                    </ol>
```